httpcore/_async/http11.py

```diff
@@ -34,6 +34,9 @@ def __init__(
 
         self.state = ConnectionState.ACTIVE
 
+    def __repr__(self) -> str:
+        return f"<AsyncHTTP11Connection state={self.state}>"
```

Comment:
Are we actually ever using this and the HTTP/2 equivalent? I _think_ the logs in the connection pool will only ever show `AsyncHTTPConnection`.

Reply:
Sure, but while debugging I find it useful to have some additional information about the connection 🙂